packages/amazonq/package.json

```diff
@@ -1,37 +1,21 @@
 {
     "name": "amazon-q-vscode",
-    "displayName": "Amazon Q (Preview) + CodeWhisperer",
-    "description": "Amazon Q (Preview) + CodeWhisperer",
-    "version": "1.0.0-SNAPSHOT",
+    "displayName": "Standalone Test",
+    "description": "",
+    "version": "0.1.0-SNAPSHOT",
     "extensionKind": [
         "workspace"
     ],
-    "publisher": "amazonwebservices",
+    "publisher": "amazonwebservices-internal",
```

Comment:
Hmm. I was expecting that we would publish under `amazonwebservices`.

I guess we could try this.  Then we need to update https://github.com/aws/aws-toolkit-vscode/blob/c4767fce31e7e914b8287ebf46e27bbe3375cad5/packages/core/src/shared/extensions.ts#L17 , and possibly other things

Reply:
This is what I saw in the mock page, I'll just change it back.